### PR TITLE
feat: vm-1215 vc-video

### PR DESCRIPTION
**Pull request by vladimir-buravlev**
# Project Task

Closes #

### CODE REVIEW

File: framework/ui/components/atoms/vc-video/vc-video.vue [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+:root {
+  --image-size-xs: 32px;
+  --image-size-s: 48px;
+  --image-size-m: 64px;
+  --image-size-l: 96px;
+  --image-size-xl: 128px;
+  --image-size-xxl: 145px;
+}
+
+$aspects: (
```
Comment (maksimzinchuk) [inline framework/ui/components/atoms/vc-video/vc-video.vue:91]: same

File: framework/ui/components/atoms/vc-video/vc-video.vue [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  3x2: 66.66%,
+);
+
+$paddings: xs, s, m, l, xl, xxl;
+
+.vc-video {
+  @apply tw-inline-block tw-relative;
+
+  @each $name, $aspect in $aspects {
+    &_#{$name} {
```
Comment (maksimzinchuk) [inline framework/ui/components/atoms/vc-video/vc-video.vue:104]: same

File: framework/ui/components/atoms/vc-video/vc-video.vue [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+export interface Props {
+  label?: string;
+  tooltip?: string;
+  aspect?: string;
+  rounded?: boolean;
+  bordered?: boolean;
+  clickable?: boolean;
+  source?: string;
+  size?: "auto" | "1x1" | "16x9" | "4x3" | "3x2" | "xs" | "s" | "m" | "l" | "xl" | "xxl";
```
[the remark this answers is not in the document]
Comment (vladimir-buravlev) [reply framework/ui/components/atoms/vc-video/vc-video.vue]: done

File: framework/ui/components/atoms/vc-video/vc-video.vue (hunk 1 above)
Comment (vladimir-buravlev) [reply framework/ui/components/atoms/vc-video/vc-video.vue]: done

File: framework/ui/components/atoms/vc-video/vc-video.vue (hunk 2 above)
Comment (vladimir-buravlev) [reply framework/ui/components/atoms/vc-video/vc-video.vue]: done

**Review by maksimzinchuk — APPROVED** [review]

**Pull request merged**